docs/Unit-tests.md

```diff
@@ -0,0 +1,30 @@
+# Unit tests guide
+
+
+## Prerequisites
+
+You must have [Rake](http://rake.rubyforge.org/) installed on your testing environment before launch unit tests, you can install it using gem.
+
+You can see all possible commands using `Rake -T`.
+
+Before launch unit tests, you have to set 3 env vars :
+
+* `TEST_MOUNTPOINT`: the svfs mountpoint (without the slash at the end).
+* `TEST_SEG_SIZE`: segmented file size value (in megabytes).
+* `TEST_NSEG_SIZE`: standard file size value.
+
+Now the test container will be created automatically.
+
+## Launch unit tests
+
+For launch unit tests, you just have to use `rake test` command after have follow the prerequisites instructions.
+
+You can also use and edit the script `unit-tests.sh` with your conf, located in [scripts](scripts) and do `source unit-tests.sh`.
+
+
+## Write unit tests
+
+In first, you must follow [contribution guidelines](CONTRIBUTING.md) of SVFS and use the latest version of go.
+
+All the tests are written in the [test directory](test), if you want add some tests you need consider the [Rakefile](Rakefile) and comment your tests like the previous written tests.
```

Comment:
if you want add => if you want to add
the previous written tests => previously written tests